docs/embeddable-uis/element/operation-approval.mdx

```diff
@@ -0,0 +1,427 @@
+## Element Customization
+
+With every new element, you will be able to adjust it to ensure the look and feel of it fits your application. 
+
+<div className="list">
+    <ul>
+        <li>
+			<span>
+                You can change the background color and the color of the button
+			</span>
+        </li>
+            <li>
+			<span>
+                You can change the title, message and button text to make it more user friendly
+			</span>
+        </li>
+    </ul>
+</div>
+
+![after-selected-um](/img/operation-approval/approval-r-afterSelectResource.png)
+
+[//]: # (TODO Add content)
+
+## Creating an iFrame
+
+In your newly created element, click the "Generate Code" button on the top of the screen
+
+![embed-code](/img/operation-approval/embed-code.png)
+
+A code snippet for the element will be generated for you.
+This will be an `<iframe>` that you can insert anywhere into your application. It will look like the code snippet below:
+
+:::note
+Note that it is crutial to select the Tennant relevant for this element for it to fuction properly.
+:::
+
+#### The dynamic values in src are:
+- <ELEMENT_NAME> - The name of the element you have created.
+- <SOME_UNIQUE_ID> - The unique id of the environment you are using.
+- <TENANT_KEY> - The tenant key you are using.
+- <RESOURCE_INSTANCE_KEY> - The resource instance key you are using or new resource instance key.
+
+Replace them with your values.
+
+```html
+<iframe
+	title="Permit Element Name"
+	src="https://embed.permit.io/<ELEMENT_NAME>?envId=<SOME_UNIQUE_ID>&darkMode=false&resourceInstanceKey=<RESOURCE_INSTANCE_KEY>&tenantKey=<TENANT_KEY>"
+	width="100%"
+	height="100%"
+	style="border: none;"
+/>
+```
+
+Once the iframe is ready, we need to embed it. To do that, you'll need to create a login. In the next section, we'll walk you through logging in. Choose a way to log in that's convenient for you:
+
+:::info IMPORTANT
+You need to generate resource instance key for each request, and assign a reviewer to this request.
+you will pass the resource instance key in the url of the iframe, for example:
+```html
+<iframe
+    title="Permit Element Name"
+    src="https://embed.permit.io/<ELEMENT_NAME>?envId=<SOME_UNIQUE_ID>&darkMode=false&tenantKey=<TENANT_KEY>&resourceInstanceKey=<RESOURCE_INSTANCE_KEY>"
+    width="100%"
+    height="100%"
+    style="border: none;"
+/>
+:::
+
+
+## New Operation Approval
+
+When a user needs to request approval for a specific action, they should provide a reason for the request and then click the "Request Approval" button to submit it.
+Once the request is submitted, the user should obtain the key of the resource instance from the new request and assign a reviewer to it.
+
+For example, if a user wants to seek approval for a $1000 transfer to another user and the key of the resource instance is "transfer-1", they should assign a relevant user with ReBAC permissions in the following format: transfer:transfer-1#_reviewer_.
```

Comment:
the _reviewer_ role display as Italic type and it is supposed to be display with _ before and after the "reviewer"